apisix/control/v1.lua

```diff
@@ -269,6 +269,13 @@ local function iter_add_get_routes_info(values, route_id)
         if new_route.value.upstream and new_route.value.upstream.parent then
             new_route.value.upstream.parent = nil
         end
+        -- remove healthcheck info
```

Comment:
we need remove healthcheck info from `iter_add_get_services_info` too

Reply:
done